db/sql/040_create_componse_events.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS compose_audit_events (
+	id SERIAL,
+	event_id CHAR(24) UNIQUE,
+	created_at TIMESTAMP,
```

Comment:
Should this have a timezone?

Reply:
Although it seems the API returns the event in GMT it won't hurt, so I changed it to "TIMESTAMPTZ"